Jenkinsfile

```diff
@@ -712,18 +712,6 @@ try {
         }
       }
     },
-    'Scala: GPU': {
```

Comment:
@nswamy @lanking520 was this intentional? I think you just disabled all unit tests for Scala on GPU ._.